.spelling

```diff
@@ -1290,6 +1290,47 @@ Microsoft.CSharp
 v7.2.10
 7.2.x
 v7.3.3
+http
+webcmdlet
+argumentexception.throwifnullorempty
+bitconverter.tostring
+convert.tohexstring
+requires.notnullorempty
+argumentoutofrangeexception.throwifnegativeorzero
+callerargumentexpression
+requires.notnull
+argumentnullexception
+throwifnull
+process.cs
+setrequestcontent
+streamhelper.cs
+invokerestmethodcommand.common.cs
+gethttpmethod
+httpmethod
+removenulls
+notnull
+argumentnullexception.throwifnull
+disable_telemetry
+langversion
+microsoft.extensions.objectpool
+microsoft.codeanalysis.analyzers
+benchmarkdotnet
+winforms
+benchmarkdotnet.extensions
+eg
+10ms
+100ns
+1s
+1.0ns
+1.1ns
+0.5ns
+1ns
+0.3ns.
+ns
+perflabtests.blockcopyperf.callblockcopy
+numelements
+system.tests.perf_string.trim_chararr
+system.tests.perf_array.arraycopy3d
```

Comment:
I don't see these words appear in the change log. Are they added by mistake?

Reply:
They were is some other README file under performance folder.

Reply:
I think the `dotnet-tools` folder under `test/perf` are intentionally skipped from our spelling check because they come from the .NET performance repo. See `msSpell.yml`: https://github.com/PowerShell/PowerShell/blob/8b964ded3626bf8dc965ebce81d3621ff457c8b1/.vsts-ci/misc-analysis/mdSpell.yml#L35

It's also skipped in our `CONTRIBUTING.md`: https://github.com/PowerShell/PowerShell/blob/8b964ded3626bf8dc965ebce81d3621ff457c8b1/.github/CONTRIBUTING.md#spellchecking-documentation

So, if these are from the `test/perf/dotnet-tools`, they should be removed.

Reply:
Need to update the instructions in the release template. I will remove these.

Reply:
fixed